.github/how-to-write-a-good-issue.md

```diff
@@ -0,0 +1,25 @@
+# How to write a good issue
+
+Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting bugs and feature requests.
+
+For other type of questions, consider using one of:
+
+- the [Traefik community Slack channel](https://traefik.herokuapp.com)
+- [StackOverflow](https://stackoverflow.com/questions/tagged/traefik)
```

Comment:
I very much like this rule. What I'm wondering though is what kind of consequences are we going to draw from this? I can see two:

- Close question-style issues right away, pointing to Slack/StackOverflow.
- Just tolerate them (and maybe leave a "please use Slack/StackOverflow next time" message).

I know the Kubernetes folks do the former. However, they have to be very rigid due to the sheer amount of issues they receive.